scheduler/src/cook/mesos/util.clj

```diff
@@ -361,28 +361,29 @@
                                (-> 1 t/hours t/in-millis)))
         entid-start (d/entid-at db :db.part/user expanded-start)
         entid-end (d/entid-at db :db.part/user expanded-end)
-        job-user-entid (d/entid db :job/user)]
-    (->> (d/seek-datoms db :avet :job/user user entid-start)
-         (take-while #(and (< (:e %) entid-end)
-                           (= (:a %) job-user-entid)
-                           (= (:v %) user)))
-         (map #(:e %))
-         (map (partial d/entity db))
-         (filter #(<= (.getTime start) (.getTime (:job/submit-time %))))
-         (filter #(< (.getTime (:job/submit-time %)) (.getTime end)))
-         (filter #(= state-keyword (:job/state %)))
-         (filter #(not (:job/custom-executor %))))))
+        job-user-entid (d/entid db :job/user)
+        jobs (->> (d/seek-datoms db :avet :job/user user entid-start)
+                  (take-while #(and (< (:e %) entid-end)
+                                    (= (:a %) job-user-entid)
+                                    (= (:v %) user)))
+                  (map #(:e %))
+                  (map (partial d/entity db))
+                  (filter #(<= (.getTime start) (.getTime (:job/submit-time %))))
+                  (filter #(< (.getTime (:job/submit-time %)) (.getTime end)))
+                  (filter #(= state-keyword (:job/state %))))]
+    (cond->> jobs
+             (not include-custom-executor?) (filter #(not (:job/custom-executor %))))))
```

Comment:
I know this was carried over from the old code, but can we change the `(filter #(not (:job/custom-executor %))` to `(remove :job/custom-executor)`?

Reply:
Done